internals/api/secret_version_test.go

```diff
@@ -44,7 +46,8 @@ func TestCreateSecretVersionRequest_Validate_MaxSize(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		id := uuid.New()
+		id, err := uuid.NewV4()
+		assert.OK(t, err)
```

Comment:
In other tests you use `Must`. Shall we pick one method and use that consistently?